Troubleshooting: Component Content is not Scrollable (#5442)
DevExpress/devextreme-documentation: concepts/80 Troubleshooting/45 Component Content is not Scrollable/25 Scrollbar is Displayed but Component is Overlapped.md

> Sometimes a component displays a scrollbar but elements below the component overlap its content. For example, your task is to place a component between a header and footer with fixed heights but this issue occurs.

Reviewer: Once again - the edits might depend on whether we are talking about overlapping the scrollbar or overlapping component content...

> Sometimes a component displays a scrollbar but elements below the component overlap the component content. For example, your task is to place a component between a header and footer with fixed heights but this issue occurs.

Reviewer: ```suggestion
Sometimes a scrollbar is available, but elements below the component overlap component content. For example, this issue may occur if you place a component between a header and a footer with fixed heights.
```
